example/lib/main.dart

```diff
@@ -163,9 +157,9 @@ class _MyHomePageState extends State<MyHomePage> {
                   displayName: '8',
                 ),
                 GanttAbsoluteEvent(
-                  displayName: 'Absoulte Date event',
-                  startDate: DateTime(2022, 6, 7),
-                  endDate: DateTime(2022, 6, 20),
+                  displayName: 'Absolute Date event',
+                  startDate: DateTime.now().subtract(const Duration(days: 7)),
```

Comment:
how should the gantt chart display an event that starts before the chart's start date ?

Reply:
true